### ISSUE TITLE
Application Hangs Fetching Detail

### ISSUE THREAD

**Issue by freshottr** [CS599-MEDLINE/pubminer-web-app#95]
The application hangs when trying to get the abstract for PubMed: 27034514 PubMed Central: 4794306. This is the last search result for `zika`.

```
fetching details for 27034514
error extracting abstract for detail document TypeError: Cannot read property 'AbstractText' of undefined TypeError: Cannot read property 'AbstractText' of undefined
    at DocumentHelper.convertXmlToJson.then.json (/Users/ismith/school/cs599/pubminer-ui-investigation/DocumentHelper.js:127:22)
    at <anonymous>
unable to get abstract for PMID 27034514 { InvalidDocumentFormatError: bad document format
    at DocumentHelper.convertXmlToJson.then.catch.err (/Users/ismith/school/cs599/pubminer-ui-investigation/DocumentHelper.js:153:23)
    at <anonymous>
  name: 'InvalidDocumentFormatError',
  severity: 0,
  cause: TypeError: Cannot read property 'AbstractText' of undefined
    at DocumentHelper.convertXmlToJson.then.json (/Users/ismith/school/cs599/pubminer-ui-investigation/DocumentHelper.js:127:22)
    at <anonymous> }
```

**Comment by freshottr** [comment]
We need to:
 - properly recover from any errors
 - instead of throwing out a error, give back something like `'No abstract for <articleID>'` or something to that effect

**Comment by freshottr** [comment]
Resolved by #98
